site2/website-next/src/theme/DocVersionBanner/index.js

```diff
@@ -25,7 +25,7 @@ function UnreleasedVersionLabel({ siteTitle, versionMetadata }) {
       }}
     >
       {
-        "This is unreleased documentation for {siteTitle} {versionLabel} version."
+        "This documentation is for an unreleased {siteTitle} {versionLabel} version of Apache Pulsar. "
```

Comment:
Seems {siteTitle} is duplicate with `Apache Pulsar`, and maybe {versionLabel} can also be removed.

Reply:
```suggestion
        "This documentation is for an unreleased version of Apache Pulsar. "
```

Agree. Simply writing "an unreleased version" works fine for me.